### DISCUSSION TITLE
If I want an application installed in the sandbox to open properly outside the sandbox, how do I set the configuration

### DISCUSSION THREAD

**Discussion by lmou523** [sandboxie-plus/Sandboxie#3066] [Documents and notes]
This is my current configuration, which unisolates files from the registry
UnfilteredToken=y
OriginalToken=y
UnrestrictedToken=y
NoSecurityIsolation=y
OpenCredentials=y
RunServicesAsSystem=y

OpenFilePath=*
OpenKeyPath=*

However, there is another problem, if the installer registers the service, the service installed in the sandbox will be redirected
What configuration should I add so that the service installed inside the sandbox is the same as that installed outside the sandbox

**Comment by bastik-1001** [comment]
I can't give you any more insight, but I am curious to why making an effort to install something in Sandboxie, just to run it outside of it.

**Comment by lmou523** [comment]
> I can't give you any more insight, but I am curious to why making an effort to install something in Sandboxie, just to run it outside of it.

This is a job that cannot be done (caused by the service) and I will close this topic
